src/service/url-replacements-impl.js

```diff
@@ -513,6 +514,21 @@ export class GlobalVariableSource extends VariableSource {
       return this.getStoryValue_(storyVariables => storyVariables.pageId,
           'STORY_PAGE_ID');
     });
+
+    this.set('CONTAINER_TOKEN', () => {
+      let fragment = getFragment(this.ampdoc.win.location.href);
```

Comment:
This needs to check `location.originalHash`, we wipe the viewer params from the `location.href` (and therefore `location.hash`) on load.

Better yet, you can use `viewer`'s `#getParam` to get these without doing a query parse.